### ISSUE TITLE
Update from 1.2.53 to 2.1.38: Nodes reconnect every few seconds

### ISSUE THREAD

**Issue by MeMichael151** [Supergiovane/node-red-contrib-knx-ultimate#256]
Hi there! I very much appreciate your work for the KNX fans out there! I've been using your Node Red Integration for a couple of years now. Just migrating from NodeRed on a RPi to NodeRed as Addon on Home Assistant (on a RPi nonetheless).

**Describe the bug**
My new setup with NodeRed as Addon in HA has the Integration at version 2.1.38 and previously on the other Pi it was running 1.2.53.
I notice now that all the KNX Ultimate Nodes appear to be reconnecting every couple of seconds.
1. PM Connected. On duty.
2. PM Next cycle will reconnect...
3. PM .... (too fast to read)
4. PM Auto reconnecting in Progress.
5. PM ... (too fast to read)

**To Reproduce**
Run NodeRed as Addon on the latest version of HA OS. Add the KNX Ultimate Integration. Add a Node and configure the Bus Interface. Run the Flow. 

**Expected behavior**
Perhaps this is as intended, but it looks odd and is different from the behavior on my RPi where the Node Status never changed, unless a Group Address was received.

**Screenshots**
![image](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/110401368/ab4536bf-f91f-4c0c-bad6-fb831fa3f196)
![image](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/110401368/91f4f338-e1f8-4645-8426-fefc0ba7ddac)

**Knx-Ultimate Version**
 - 2.1.38

**Are you running node-red behind homematic, docker or anything similar?**
 - NodeRed as Addon on Home Assistant OS on a Raspberry Pi

Appreciate your look into this. As said, perhaps this is as you intended it. 
Best, Michael

**Comment by Supergiovane** [comment]
Hi Michael!
This is not the intended behaviour. You should have some issues.
Do you maybe have reached the maximum allowed connection of your KNX/IP Interface?
Please check that by turning off your raspberry, restart homeassistant and also the KNX/IP Interface.
If unsuccessful, i need you to set the log level to "debug" (in the gateway config node) and then post the log here (a screenshot is also OK).
Thanks.

**Comment by MeMichael151** [comment]
Thanks for getting back to me on this. HA does provide a log:

[error] 8/2/2023, 10:26:24 AM.245 knxUltimate-config: received KNXClientEvents.error: addMembership EINVAL
[error] 8/2/2023, 10:26:24 AM.249 knxUltimate-config: Disconnected by: addMembership EINVAL
[error] 8/2/2023, 10:26:32 AM.246 KNXUltimate-KNXEngine: Multicast: cannot add membership (Error: addMembership EINVAL
    at Socket.addMembership (node:dgram:852:11)
    at Socket.<anonymous> (/config/node-red/node_modules/node-red-contrib-knx-ultimate/KNXEngine/src/KNXClient.js:179:30)
    at Socket.onListening (node:dgram:255:7)
    at Socket.emit (node:events:526:35)
    at startListening (node:dgram:181:10)
    at node:dgram:369:7
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -22,
  code: 'EINVAL',
  syscall: 'addMembership'
})

This is after I restarted the Bus. For a short period yesterday evening I had Home Assistant, NodeRed on Home Assistant and NodeRed on another RPi connecting to the KNX Interface. But the other RPi was shutdown after everything was moved over to the HA-NodeRed last night. Also I restarted HomeAssistant now. 

I set the log level to "debug" but wasn't sure where to find the Log itself. Did you mean this, also in the Gateway Config Node:
![image](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/110401368/962982c8-63c9-4efb-b20f-6ee80f6bfd6b)
![image](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/110401368/30775e13-b972-4bf0-b1ac-7f24588fcf69)

Thanks! This worked before with HA and NodeRed on separate RPis. I am somewhat confused why it shouldnt work now anymore.
